.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,6 +10,11 @@ assignees: ''
 **Describe the bug**
 A clear and concise description of what the bug is.
 
+**App versions**
+Android:
```

Comment:
As this for is mainly used for bugs in the web UI, perhaps we don't need Android in the template?

Also, I just realized we don't actually have build numbers / versions in the web UI, only in Android. Perhaps we should add an FR to show the version in the about screen before adding this to the template?

Reply:
Filed https://github.com/google/ground-platform/issues/2015

I think it would be useful to include both when applicable, since a mismatch of versions could also be a root cause of bugs, or there could be a cross-app interaction that needs debugging. WDYT?